patches/vitepress+1.0.0-rc.43.patch

```diff
@@ -0,0 +1,107 @@
+diff --git a/node_modules/vitepress/dist/client/app/router.js b/node_modules/vitepress/dist/client/app/router.js
+index e1f068e..0ad3bb0 100644
+--- a/node_modules/vitepress/dist/client/app/router.js
++++ b/node_modules/vitepress/dist/client/app/router.js
+@@ -26,7 +26,8 @@ export function createRouter(loadPageModule, fallbackComponent) {
+         await router.onAfterRouteChanged?.(href);
+     }
+     let latestPendingPath = null;
+-    async function loadPage(href, scrollPosition = 0, isRetry = false) {
++    async function loadPage(href, scrollPosition = 0, isRetry = false, alreadyTriedLoadingRootFallback = false) {
```

Comment:
This is going to be a fucking nightmare to maintain, and fix when vitepress updates.

Reply:
Not really